### ISSUE TITLE
YouTube video player is cropped on the right side when Dash to Dock is activated

### ISSUE THREAD

**Issue by cosmic-echo** [micheleg/dash-to-dock#797]
https://imgur.com/a/3vTxRXV

The button to expand the video frame vanished. I cleaned the cache already, but it doesn't make a difference. It's been like that for days. But it's also weird, because the fullscreen button appears while the page is being loaded. I can see the scrollbar on the right disappearing for a moment then appearing again.

The workaround for this is to unmaximize the window and maximize it again or disable the extension temporarily.

The bug is also present in Firefox and Chromium even after a fresh install of Ubuntu 18.04 LTS. I installed Dash to Dock from the gnome-shell-extension-dashtodock package, but the version from extensions.gnome.org has the same issue. I'm not sure if it's a bug from the extension or from YouTube itself, but it's weird that disabling Dash to Dock fixes the issue.

**Comment by dvorapa** [comment]
Yeah, I experienced this issue as well, but since the last version, the issue is gone. Does this still happen to you?

**Comment by dvorapa** [comment]
Or at least now they hide for a while while loading, but then show up again
